cypress/e2e/thirdTest.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Third Test', () => {
```

Comment:
pls, use a more meaningful name to describe. It can be Login functionality, or etc